mesa/space.py

```diff
@@ -778,11 +778,120 @@ def out_of_bounds(self, pos):
 
 class NetworkGrid:
     """ Network Grid where each node contains zero or more agents. """
+    def __init__(self, G=None, generator="Graph", args=()):
+        """Default values create empty graph"""
+        if G is not None:
+            self.G = G
+        elif generator is not None:
+            if type(args) is tuple:
```

Comment:
I think the recommended way for type checks is
`isinstance(args, tuple)`
You might want to consider changing this here and below. 
Alternatively you might just as well skip the check and just use a `try...except` statement.
This comment applies multiple times below

Reply:
I think try/except won't work (least not everywhere) because e.g. anything can be a node in networkx graphs.

But I take on board the isinstance comment, it makes sense that anything inheriting from those types could be valid too.

Reply:
Ok I changed that in my branch, is that all I need to do?

Reply:
I can't see your changes, maybe you forgot to push your local changes to your forked repo?

Reply:
herp derp no, forgot to add changed file!